src/develop.js

```diff
@@ -91,32 +91,6 @@ const List = () => {
       localStorage.setItem('list', JSON.stringify(empty));
```

Comment:
- [ ] **Required:** The Crude operations have been implemented perfectly 🎉 . However, The status of the data is not persisted. I see that the deleted Items work (When they're deleted individually) - But, the items marked as completed to do stay persisted after refreshing. See attached video clip for your reference.

[Watch this video demonstrating the issue](https://user-images.githubusercontent.com/35315311/178086339-b341e96b-13c8-40a0-8e26-c6d70700b569.webm)

- [OPTIONAL]: As a boost to your APP 🚀 , you can also consider implementing the above feature to a `Clear All completed` button as well.